app/interfaces/api/entities/ccr_claim.rb

```diff
@@ -104,16 +114,45 @@ def offence_code_id
         ('A'..'K').zip(501..511).to_h[offence_class_code]
       end
 
-      def wrapped_bill
-        [{
+      def number_of_witnesses
+        object.fees.find_by(fee_type_id: 10)&.quantity&.to_i || 0
```

Comment:
The `object.fees.find_by(fee_type_id: `xx`)&.quantity&.to_i` is repeated multiple times, this could be extracted to a method for re-use.  

Also, as discussed, using constants for the fee_type_ids might make reading the code easier e.g. 
```ruby
WITNESSES = 10
CASES = 9
def count_of(fee_type)
  object.fees.find_by(fee_type_id: fee_type)&.quantity&.to_i
end
```

```ruby
count_of(CASES)
count_of(WITNESSES)
```